src/System Application/App/Language/src/LanguageImpl.Codeunit.al

```diff
@@ -151,18 +152,40 @@ codeunit 54 "Language Impl."
 
     procedure GetApplicationLanguages(var TempWindowsLanguage: Record "Windows Language" temporary)
     var
+        SupportedLanguage: Record "Supported Language";
         WindowsLanguage: Record "Windows Language";
+        LanguageFilter: Text;
     begin
+        SupportedLanguage.Reset();
+        SupportedLanguage.ReadIsolation := IsolationLevel::ReadUncommitted;
```

Comment:
Do we need uncommitted data? 
I'd assume `ReadCommitted` is sufficient.

Reply:
This was made to ensure no locking on a system table - which is also a default read behavior in BC unless otherwise is specified in code.

I will await full feedback including valid Fixed # refererence mentioned here: https://github.com/microsoft/BusinessCentralApps/issues/1061#issuecomment-2519896200

My to-do list as I see it:

- Name change to "Available Language" (see comment from Søren
- Teaching Tips
- Breaking code lines as you requested
- Isolation level

..and then I need some Object no's assigned

Reply:
Sounds good.

Use ID 3563 for page and table. I just checked that isn't used for any page or table..